js/ui/list/ui.list.base.js

```diff
@@ -746,8 +747,17 @@ export const ListBase = CollectionWidget.inherit({
             .addClass(LIST_GROUP_CLASS)
             .appendTo(this._itemContainer());
 
+        const id = `dx-${new Guid().toString()}`;
```

Comment:
.toString() shouldn’t be used here.

Reply:
It's hard to read and understand without .toString()